content/en/docs/Writing policies/mutate.md

```diff
@@ -359,6 +359,10 @@ Because these mutations occur on existing resources, Kyverno may need additional
 
 To define such a policy, trigger resources need to be specified in the `match` block. The target resources--resources that are mutated in the background--are specified in each mutate rule under `mutate.targets`. Note that all target resources within a single rule must share the same definition schema. For example, a mutate existing rule fails if this rule mutates both `Pod` and `Deployment` as they do not share the same OpenAPI V3 schema (except `metadata`).
 
+{{% alert title="Note" color="warning" %}}
+Keep in mind that mutations to some immutable fields by Pod update operations like patch, and replace have [some limitations](https://kubernetes.io/docs/concepts/workloads/pods/#pod-update-and-replacement).
```

Comment:
I suggest the following text here:

"Mutation of existing Pods is limited to mutable fields only. See the Kubernetes documentation here for more details."